php/basicphp/controllers/AppController.php

```diff
@@ -1,9 +1,26 @@
-<?php
-
-class AppController
-{
-    public function index()
-    {
-        echo '';
-    }
-}
+<?php
+
+/**
+ * The AppController is a Class Controller reserved for endpoint
+ * routes, i.e. REST endpoints, using the route_class() function.
+ */
+
+class AppController
+{
+
+	public function index()
+	{
+		echo '';
+	}
+
+	public function viewUser()
+	{
+		echo url_value(2);

```

Comment:
does not return user id => https://travis-ci.com/the-benchmarker/web-frameworks/jobs/247934237#L508

Reply:
Apparently, $_SERVER['REDIRECT_URL'] is only applicable in Apache server... had to revert to request_uri.

Reply:
I have a `404` for **POST** on `/user`